packages/main/test/pages/Input.html

```diff
@@ -332,6 +332,11 @@ <h3>Input in a Dialog</h3>
 
 	<h3>Short Input With Centered Text</h3>
 	<ui5-input class="short" value="244"></ui5-input>
+	<ui5-input id="change-event-value" show-suggestions>
```

Comment:
This input field is just pasted in the middle of the test page next to some input. Please be consistent and add it somewhere separately and add a label or a title on top of it to indicate what could be tested here.